script.js

```diff
@@ -63,6 +63,7 @@ function QuickWiki() {
             
           if(diff > 0) {
             top = top - diff - 20;                  //extra padding of 20
+            if(top<0) top = 5;                      //adjust top position if exceeds viewport from top  
```

Comment:
This isn't related to the jQuery update, right? Don't change this now, but you could've committed this separately. Each commit should follow the Unix philosophy: do one thing, and do it well. This is so that it's easy to revert problematic commits without affecting other changes (e.g., you might end up reverting some unrelated bug fixes, unintentionally), and so it's easy to understand the project history. Check out these [guidelines for git commits][1]. `git add -p` is especially powerful for cases like the current one (basically it lets you stage _parts_ of your diff rather than the entire file at once).

Also, once you're comfortable with git's basics, you could start using [`git rebase -i`][2] to amend, squash, or re-order commits, making sure they are well-formed before you publish them. That way you can make commits quickly when you're in the thick of coding without worrying too much and clean up later.

Don't feel pressured to implement all this in your workflow immediately, but try to keep moving towards that end goal. It'll help a lot in the long run, I think.

PS: I hope you don't mind me lecturing so much :stuck_out_tongue_winking_eye:

[1]: http://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project
[2]: https://www.atlassian.com/git/tutorials/rewriting-history/git-rebase-i